admin_workspace_integration_test.go

```diff
@@ -93,6 +93,7 @@ func TestAdminWorkspaces_ListWithSort(t *testing.T) {
 		require.NoError(t, err)
 		require.NotEmpty(t, wl.Items)
 		require.GreaterOrEqual(t, len(wl.Items), 2)
+		assert.True(t, wl.Items[1].CurrentRun.CreatedAt.After(wl.Items[0].CurrentRun.CreatedAt))
```

Comment:
OK I think this test will fail in the nightly CI run. You had no way of knowing this, but we skip this test in PR actions because it is a Terraform Enterprise (skipUnlessEnterprise above) but run them against a simulated TFE instance at night.

I think this will have the same problem as below: the CreatedAt times will be zero values unless you include WSCurrentRun